xarray/tests/test_dataarray.py

```diff
@@ -1660,9 +1660,22 @@ def test_dataset_math(self):
 
     def test_stack_unstack(self):
         orig = DataArray([[0, 1], [2, 3]], dims=['x', 'y'], attrs={'foo': 2})
+        assert_identical(orig, orig.unstack())
+
         actual = orig.stack(z=['x', 'y']).unstack('z').drop(['x', 'y'])
         assert_identical(orig, actual)
 
+        dims = ['a', 'b', 'c', 'd', 'e']
+        orig = xr.DataArray(np.random.rand(1, 2, 3, 2, 1), dims=dims)
+        stacked = orig.stack(ab=['a', 'b'], cd=['c', 'd'])
+
+        for dim in (None, ['ab', 'cd'], ['ab', 'cd', 'e']):
+            unstacked = stacked.unstack(dim=dim)
+            roundtripped = unstacked\
```

Comment:
another minor point: please use an extra set of parentheses rather than writing `\` for line continuations. This is preferred by PEP8 (and looks a little cleaner, in my opinion).

Reply:
I agree that is cleaner.